### DISCUSSION TITLE
How do I use this without a bundler?

### DISCUSSION THREAD

**Discussion by douglas-legulas** [postcss/postcss#1777] [Q&A] [answered]
My employer doesn't let me use a bundler such as Webpack for our web apps (browser environment only), so what are my options?

I do use Dart Sass to compile .scss files into .css. The built files are committed to the git repository so they don't need to have Sass installed on the server.

**Comment by ai** [comment, marked as answer]
You can use `postcss-cli` to process `.css` files
